widgetssdk/src/main/java/com/glia/widgets/entrywidget/EntryWidgetContract.kt

```diff
@@ -5,6 +5,12 @@ import com.glia.widgets.base.BaseView
 
 internal interface EntryWidgetContract {
 
+    enum class ViewType {
+        BOTTOM_SHEET,
+        EMBEDDED_VIEW,
+        LIVE_SUPPORT
```

Comment:
I expected to see only two types here, without `LIVE_SUPPORT`. What is the `LIVE_SUPPORT` for?

Reply:
For SCv2

Reply:
Maybe we can use another name for it, or I can add the comments.
@DavDo, what do you think? Does this name make sense in SC?

Reply:
It sounds good, but maybe to point out the place it will be used it's better to have smth like "SC_CHAT".

Reply:
In my opinion, b) is the best one.

Reply:
Fixed 081102e951c1d9b572ec4e50f354146281c78f4e